### DISCUSSION TITLE
overrideConfig interfering with additional parameters in conversation chain

### DISCUSSION THREAD

**Discussion by tremella** [FlowiseAI/Flowise#1791] [Q&A]
I've got a chat flow which is accessed through an app. It holds a conversation just fine, but we ran into a problem where it wasn't remembering things. We solved this with an overrideConfig hash containing the sessionID, but I've noticed that unless this hash is commented out, the bot on our user interface will now totally ignore any Additional Params I put into the conversation chain node (e.g, "append &&& to the end of all responses")

This is a problem because right now there's no prompt being given on the user interface (~~IDK why the spec didn't include it~~ < now I do)

I can see that there's a Node called Chat Bot Template, which could help, but I don't really understand the Human Message part of it. I'm a newbie to this. Can anyone either suggest a fix, or explain how I should use this Node? 

Thanks in advance :pray: 
![Screenshot from 2024-02-22 11-54-00](https://github.com/FlowiseAI/Flowise/assets/63588842/8a46014f-5020-46ab-aed4-20a764a0666a)

**Comment by Jaredude** [comment]
Are you able to share a screenshot of the flow that's having the issue? I believe there was an open PR awhile back for doing a merge of overrideConfig instead of full object replacement, but if this is behaving as you're saying it must not be merged yet.

**Reply by tremella** [reply]
Hi, thanks for your response! Here's a snapshot, please forgive my use of JSPaint.
![Screenshot from 2024-02-23 11-25-29](https://github.com/FlowiseAI/Flowise/assets/63588842/e0422302-d7c4-4b5d-819d-2df2f6fef7c1)

**Reply by Jaredude** [reply]
This could be a couple of things...
- could be related to markdown and the characters appending being processed as markdown that becomes invisible in the chat client. If you make the call via Postman, does it append the characters?
- could be related to the model choosing to ignore your instructions. Try playing with the temperature and the prompt. Sometimes, just telling the model that it's important that it perform a specific response will help.

I've made a similar flow that produces the outcome you're looking for, so one of the above two are likely the culprit. Here's a screenshot of the output using Postman:
<img width="1016" alt="Screenshot 2024-02-24 at 9 18 02 AM" src="https://github.com/FlowiseAI/Flowise/assets/10228950/52ab6db8-0a9a-4d3b-9f04-98a98f8caa35">
<img width="1034" alt="Screenshot 2024-02-24 at 9 18 33 AM" src="https://github.com/FlowiseAI/Flowise/assets/10228950/e920fa09-5a9f-4777-9f17-dcfe114af077">
